core/__tests__/ops/app.ts

```diff
@@ -0,0 +1,19 @@
+import { helper } from "@grouparoo/spec-helper";
+import { api } from "actionhero";
+import { App } from "../../src";
+import { AppOps } from "../../src/modules/ops/app";
+
+describe("AppOps", () => {
+  helper.grouparooTestServer({ truncate: true, enableTestPlugin: true });
+  it("should always return a connection object when given a connect method, even when run multiple times", async () => {
+    const app = new App({
+      name: "test app",
+      type: "test-plugin-app",
+    });
+    await app.save();
+
+    for (let i = 0; i < 8; i++) {
+      expect(await AppOps.connect(app)).toBe(true);
+    }
```

Comment:
I half-agree. However the root cause of the issue is simply existing connections are not returned. So this test fails if that's the case.
I'm not sure we _need_ to do deeper testing since, if the connection is not returned, this is not true.

Reply:
That’s a good point! Ship it!